.github/actions/warm-up-repo/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Warm-up repo
```

Comment:
Could we a bit more specific on what is warmed up? Or even rename it to match the description in the next line a bit more?

Reply:
Yeah that’s a good shout – _warming up_ may mean different things if we use both Rust and Yarn. I’d stick with this action name for now just to simplify our monorepo migration, but I agree with you. Let’s revise revise what we do within a couple of weeks, once we’ve got all CI workflows in 🙂